.travis/klee.sh

```diff
@@ -208,6 +208,20 @@ else
 
   set +e # We want to let all the tests run before we exit
```

Comment:
I've just noticed there's a bug here. `set +e` is already set above so this line can be dropped.